message_ix_models/project/investment/__init__.py

```diff
@@ -0,0 +1 @@
+"""Runs with cost of capital in MESSAGE."""
```

Comment:
I feel the top-level module name `message_ix_models.project.investment` might lead to confusion in the future if there are multiple projects that deal with investment (seems likely). Some ways to disambiguate would be to add a two-digit year to the module name:
- `message_ix_models.project.coc_25`
- `message_ix_models.project.investment_25`

The choice of a CLI entry point **mix-models coc** points to the former.